README.md

```diff
@@ -52,6 +52,10 @@ React Native date & time picker component for iOS and Android
     - [Android](#android)
   - [Running the example app](#running-the-example-app)
 
+## Requirements
+
+- Xcode >= 11; please open a PR if you need to support building with older Xcode versions - see #133. Please note that we support iOS >= 10 but you need Xcode 11 to build.
```

Comment:
```suggestion
- Xcode >= 11; please open a PR if you need to support building with older Xcode versions - see [#133](https://github.com/react-native-community/datetimepicker/issues/133). Please note that we support iOS >= 10 but you need Xcode 11 to build.
```

the link actually needs to be there, sorry :D

Reply:
Done